neo/Network/P2P/Payloads/ConsensusPayload.cs

```diff
@@ -118,7 +118,7 @@ public bool Verify(Snapshot snapshot)
         {
             if (BlockIndex <= snapshot.Height)
                 return false;
-            return this.VerifyWitnesses(snapshot);
+            return this.VerifyWitnesses(snapshot, 1_00000000);
```

Comment:
Maybe we should require for be a valid CN, a script wallet of less than 1/20 or 1/50 of gas